.ast-grep/multicast-rule-validate-method-docstring.yml

```
@@ -0,0 +1,51 @@
+            kind: block
+            inside:
+              kind: function_definition
+              inside:
+                kind: block
+                inside:
+                  kind: class_definition
+      - kind: string
+  cep7_docstring_tests:
+    all:
+      - any:
+        - matches: function_docstring
```

yamllint (1.35.1)

[error] wrong indentation: expected 10 but found 8

(indentation)